laboratory-service/src/main/java/com/raf/si/laboratoryservice/dto/response/order/ResultResponse.java

```diff
@@ -0,0 +1,21 @@
+package com.raf.si.laboratoryservice.dto.response.order;
+
+import com.raf.si.laboratoryservice.model.AnalysisParameter;
+import com.raf.si.laboratoryservice.model.AnalysisParameterResult;
+import com.raf.si.laboratoryservice.model.LabAnalysis;
+import com.raf.si.laboratoryservice.model.LabWorkOrder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+public class ResultResponse {
+    LabWorkOrder order;
+    List<AnalysisParameterResult> resultList;
```

Comment:
Ako vracas listu AnalysisParameterResult klase, automatski ce se povuci i work order i analysisParameter koij ce se ubaciti u response, a povlacenjem AnalysisParameter klase ce se povuci i parameter i analysis objekti, a sve to je nepotrebno. Najbolje je da napravis jedan Dto u koji u mapperu mozes da postavis samo ono sto ti treba